sui/deploy-test.js

```diff
@@ -42,8 +46,10 @@ async function processCommand(config, chain, options) {
 
     await broadcast(client, keypair, tx);
 
+    const channelId = await getChannelId(client, singleton.objectId);
```

Comment:
move deployment to deploy-contract

Reply:
I think it's better to be reviewed as a separate pr?

Reply:
created a ticket here https://axelarnetwork.atlassian.net/browse/AXE-4528